password.js

```diff
@@ -0,0 +1,150 @@
+const crypto = require('crypto');
```

Comment:
I'm paranoid, but we should add a `'use strict'`  at the top here.

Reply:
Umm, sure.